tests/unit/test_queries.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+import unittest
+
+import boto3
+from botocore.exceptions import ClientError
+from opensearchpy import OpenSearch, RequestsHttpConnection, AWSV4SignerAuth
+import pytest
+import time
+from sds_in_a_box.SDSCode import queries
+from sds_in_a_box.SDSCode.opensearch_utils.action import Action
+from sds_in_a_box.SDSCode.opensearch_utils.index import Index
+from sds_in_a_box.SDSCode.opensearch_utils.document import Document
+from sds_in_a_box.SDSCode.opensearch_utils.payload import Payload
+from sds_in_a_box.SDSCode.opensearch_utils.client import Client
+
+
+@pytest.mark.network
+class TestQueries(unittest.TestCase):
+
+    def setUp(self):
+        #Opensearch client Params
+        os.environ["OS_DOMAIN"] = 'search-sds-metadata-uum2vnbdbqbnh7qnbde6t74xim.us-west-2.es.amazonaws.com'
```

Comment:
Is this specific to your stack?

We might want to look at mocking opensearch in the future: https://pypi.org/project/openmock/

Reply:
This is a test domain that I've been using for my unit tests. 
Opened issue https://github.com/IMAP-Science-Operations-Center/SDS-in-a-box/issues/24 as a result of this comment.